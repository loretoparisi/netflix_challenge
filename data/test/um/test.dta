1 79 1123 2
1 144 1349 3
1 149 1206 4
1 191 1845 4
1 211 918 5
1 463 1145 3
1 614 918 3
1 683 1580 5
1 701 1349 4
1 849 2056 5
1 870 2056 4
1 889 1587 5
1 892 918 3
1 979 939 5
1 1006 2056 5
1 1103 1349 5
1 1312 1809 4
1 1333 1105 4
1 1381 1123 5
1 1394 918 0
1 1432 2056 4
1 1529 2056 4
1 1632 2056 4
1 1819 1109 5
1 1927 1106 5
1 1942 1349 0
1 1983 2235 5
1 2189 1469 5
1 2193 1136 4
1 2299 1539 5
1 2300 2056 4
2 2349 1106 5
2 2378 918 5
2 2589 1359 5
2 2647 918 5
2 2664 2235 5
2 2709 2056 4
2 3046 918 3
2 3053 1580 4
2 3059 1206 5
2 3080 918 5
2 3111 1105 4
2 3127 2235 3
2 3150 1136 5
2 3181 1349 5
2 3187 1136 0
2 3221 1801 5
2 3233 2235 3
3 3317 2235 1
3 3578 2235 0
