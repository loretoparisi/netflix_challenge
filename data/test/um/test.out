4.241
4.241
4.438
1
